src/applications/vaos/appointment-list/components/cards/confirmed/ConfirmedAppointmentListItem.jsx

```diff
@@ -107,12 +115,25 @@ export default function ConfirmedAppointmentListItem({
         {header}
       </div>
       <h3 className="vaos-appts__date-time vads-u-font-size--h3 vads-u-margin-x--0">
+        {isAtlasLocation(appointment) && (
+          <span>Video appointment {getVideoLocation()}</span>
```

Comment:
Do we plan on adding any additional logic to `getVideoLocation()`?

If not, you could possibly remove that function and make this a little more concise by doing: 

```js
<span>
  Video appointment{isAtlasLocation(appointment) && ' at an ATLAS location'}
</span>
```

Reply:
Yes, additional logic will be added to support video appointments at home, VA location, and VA device